Feat/about conference

DagmaraSobczak opened the pull request.

DagmaraSobczak committed:
Added images

DagmaraSobczak committed:
Added images

DagmaraSobczak committed:
Added images

DagmaraSobczak committed:
Added AboutConference component

DagmaraSobczak committed:
Added AboutConference component

DagmaraSobczak committed:
Added AboutConference component

DagmaraSobczak committed:
added CSS styles and content to the 'About the Conference' section

DagmaraSobczak committed:
added CSS styles and content to the 'About the Conference' section

DagmaraSobczak committed:
added CSS styles and content to the 'About the Conference' section

DagmaraSobczak committed:
fixed css style "about conference" section

DagmaraSobczak committed:
fixed css style "about conference" section

DagmaraSobczak committed:
fixed css style "about conference" section

DagmaraSobczak committed:
fixed css style "about conference" section

DagmaraSobczak committed:
added CSS styling and content for the 'What we want to achieve' section

DagmaraSobczak committed:
added CSS styling and content for the 'What we want to achieve' section

DagmaraSobczak committed:
added CSS styling and content for the 'What we want to achieve' section

barteknr1 commented on src/components/AboutConference/AboutConference.jsx:
```diff
@@ -0,0 +1,63 @@
+            alt=""
+          />
+        </div>
+        <button className={css.AboutConferenceButton}>Więcej</button>
```
Do buttona jest gotowy komponent. Musisz mieć aktualną wersję z dev

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+  font-size: 31px;
+  color: #000000;
+  font-weight: 400;
+  line-height: auto;
```
line-height nie może mieć wartości auto

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+.AboutConferenceTitle {
+  font-family: var(--decoraction-font);
+  font-size: 31px;
+  color: #000000;
```
Mamy zmienną na czarny kolor:   --dark-text-color: #1d1d1d;

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+
+.AboutConferenceTitle {
+  font-family: var(--decoraction-font);
+  font-size: 31px;
```
Wielkość nagłówków w figmie wynosi 39px

barteknr1 commented on src/components/AboutConference/AboutConference.jsx:
```diff
@@ -0,0 +1,63 @@
+      <div className={css.container}>
+        <div className={css.AboutConference}>
+          <div className={css.AboutConferenceWraper}>
+            <h className={css.AboutConferenceTitle}>O konferencji</h>
```
h nie może być użyte bez cyferki. Tutaj najodpowiedniejsze będzie h3. Analogicznie również trzeba zmienić pozostałe.

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+  font-family: var(--decoraction-font);
+  font-size: 31px;
+  color: #000000;
+  font-weight: 400;
```
400 jest wartością nominalną dla `<p>`, więc można tego nie pisać

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+    flex-direction: row;
+    max-width: 1105px;
+    max-height: 365px;
+    flex-wrap: nowrap;
```
Nie znalazłem efektu nowrap. Czy jest jakiś powód, dla którego tu jest?

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+  flex-direction: column;
+
+  width: 363px;
+  height: 1943px;
```
Wysokości kontenera nie można ograniczać. Musi ona wynikać z zawartości

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+@media screen and (min-width: 765px) {
+  .container {
+    width: 1064px;
+    height: 1100px;
```
Jak wyżej

barteknr1 commented on src/components/AboutConference/AboutConference.jsx:
```diff
@@ -0,0 +1,63 @@
+          <img
+            className={css.AboutConferenceImg2}
+            src="src\assets\images\about_foto2.png"
+            alt="ludzie siedzący przy stolikach"
```
Tutaj trzeba zmienić alt :)

barteknr1 commented on src/components/AboutConference/AboutConference.jsx:
```diff
@@ -0,0 +1,63 @@
+          <img
+            className={css.AboutConferenceVector}
+            src="src\assets\images\about_Vector1.png"
+            alt=""
```
+1 za pusty alt 👍

barteknr1 commented on src/components/AboutConference/AboutConference.jsx:
```diff
@@ -0,0 +1,63 @@
+              Konferencja „Uwaga! Smartfon” to unikatowe w skali całej Europy
+              wydarzenie poświęcone zagadnieniom cyfryzacji i nadmiernemu
+              korzystaniu z urządzeń ekranowych przez dzieci i młodzież.
+              <br></br>
```
br jest tutaj zbędne.

barteknr1 commented on src/components/AboutConference/AboutConference.jsx:
```diff
@@ -0,0 +1,63 @@
+        </div>
+        <div className={css.AboutConference2}>
+          <div className={css.AboutConferenceWraper2}>
+            <h className={css.AboutConferenceTitle2}>Co chcemy osiągnąć?</h>
```
h3

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+  }
+}
+
+.AboutConferenceButton {
+  width: 32px;
+  height: 16px;
+  border-radius: 16px;
+  background-color: var(--primary-background-color);
+  border-color: var(--dark-text-color);
+  color: var(--dark-text-color);
+}
```
Do usunięcia, będziesz miała gotowy komponent.

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+  line-height: 24px;
+  letter-spacing: 0.12px;
+  font-weight: 400;
+  margin-left: 20px;
```
Ten margines powinien być w kontenerze

barteknr1 commented on src/components/AboutConference/AboutConference.module.css:
```diff
@@ -0,0 +1,218 @@
+  z-index: -1;
+  position: absolute;
+  bottom: 30px;
+  left: 0;
+  right: 0;
```
Jedno z nich wyklucza drugie

barteknr1 reviewed (changes requested):
Cześć, z tego co widzę to masz przygotowane obrazy, ale nie zrobiłaś trzeciego segmentu. Dodaj go proszę :)
Postaraj się nie ograniczać za pomocą height niczego co zawiera tekst oraz obrazków. Generalnie wysokość większości rzeczy powinna wynikać właśnie z zawartości.
Zauważyłem, że masz powtórzone style np. .AboutConferenceTitle2, .AboutConferenceText2. itd. Wszystkie segmenty powinny mieć wspólne style, więc te z numerem 2 nie są tak na prawdę w ogóle potrzebne.
Ogólnie masz trochę niedociągnięć w stylach. Czy potrzebujesz pomocy?

DagmaraSobczak commented:
hej, jeszcze nie jest gotowe. wrzuciłam część żeby takiej dużej porcji nie puścić na raz.

DagmaraSobczak committed:
added CSS styling and content for the 'Whom do we want to engage?' section

DagmaraSobczak committed:
added CSS styling and content for the 'Whom do we want to engage?' section

DagmaraSobczak committed:
added CSS styling and content for the 'Whom do we want to engage?' section

DagmaraSobczak committed:
added CSS styling and content for the 'Whom do we want to engage?' section

PiotrDrechsler committed:
feat/ mobile navbar

add media for mobile resolutions

barteknr1 committed:
add basic modal styles

barteknr1 committed:
feat/basic-modal

PiotrDrechsler committed:
fix/ NavLink

Change Link to NavLink

PiotrDrechsler committed:
fix /icons

add hamburger-menu to icons

PiotrDrechsler committed:
fix/ SharedLayout

delete button - unused import

DagmaraSobczak committed:
Update Home.jsx

DagmaraSobczak committed:
Update Home.jsx

DagmaraSobczak committed:
Update Home.jsx

DagmaraSobczak committed:
changed button

DagmaraSobczak committed:
changed button

DagmaraSobczak committed:
Merge branch 'feat/AboutConference' of https://github.com/barteknr1/uwaga-smartfon into feat/AboutConference

PiotrDrechsler committed:
fix/ mobile-navbar

1.RWD styles for mobile and desktop navbar
2. Hamurger menu for mobile
3. Toggle acrtion to show mobile menu on mobile devices

PiotrDrechsler committed:
Merge branch 'dev' into feat/AboutConference

PiotrDrechsler committed:
fix/ Navigation

delete an unnecessary import

PiotrDrechsler committed:
fix/ navigation styles

delete rem add px for font size

PiotrDrechsler committed:
Merge pull request #26 from barteknr1/feat/mobile-navbar

Feat/mobile navbar

barteknr1 committed:
Merge branch 'dev' into feat/modal

barteknr1 committed:
Merge pull request #22 from barteknr1/feat/modal

Feat/modal

barteknr1 committed:
feat/modal window closing

barteknr1 committed:
Merge pull request #27 from barteknr1/feat/modal-closing

feat/modal window closing

DagmaraSobczak committed:
Corrected CSS style

DagmaraSobczak committed:
Corrected CSS style

DagmaraSobczak committed:
Merge branch 'feat/AboutConference' of https://github.com/barteknr1/uwaga-smartfon into feat/AboutConference

DagmaraSobczak committed:
Merge branch 'feat/AboutConference' of https://github.com/barteknr1/uwaga-smartfon into feat/AboutConference

DagmaraSobczak committed:
changed position and z index
